README.md

```diff
@@ -2,42 +2,6 @@
 
 This repository holds the metadata for Keptn integrations to be listed on the [ArtifactHub](https://artifacthub.io). Each Keptn integration listed needs to have an [`artifacthub-pkg.yml` file](https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml) and a `README.md` file.
 
-## `artifacthub-pkg.yml` structure
+To get listed on the [Keptn docs integration page](https://keptn.sh/docs/integrations/), you must raise a PR here.
```

Comment:
```suggestion
To get listed on the [Keptn docs integration page](https://keptn.sh/docs/integrations/), you must raise a PR in this repository.
```

Reply:
Should this also mention how to get ones integration listed under the correct category (e.g. Deployment, Observability, Webook).  I'm not sure that is being done properly for all integrations and it will become increasingly important as more integrations are added.